hoodie-client/src/test/java/com/uber/hoodie/table/TestMergeOnReadTable.java

```diff
@@ -83,7 +83,7 @@
 
 public class TestMergeOnReadTable {
```

Comment:
How about testing this with the embeddedTimelineServer ?

Reply:
Added.